nil/internal/collate/syncer.go

```diff
@@ -321,7 +321,7 @@ func (s *Syncer) saveBlock(ctx context.Context, block *types.BlockWithExtractedD
 			Uint64(logging.FieldBlockNumber, uint64(block.Id)).
 			Stringer(logging.FieldBlockHash, block.Hash(s.config.ShardId)).
 			Stringer(logging.FieldShardId, s.config.ShardId).
-			Stringer(logging.FieldSignature, block.Signature).
+			Stringer(logging.FieldSignature, &block.Signature).
```

Comment:
Do we really need it? If `*Signature` is a Stringer then `Signature` should be also Stringer